From a98b420a5fe3901a38fa9512c9ba3aaba536c898 Mon Sep 17 00:00:00 2001
From: Eric Squires <eric.g.squires@gmail.com>
Date: Thu, 4 Feb 2021 09:30:28 -0500
Subject: [PATCH] add ccache

---
 CMakeLists.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e52750ace..6ae7919ba 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,6 +3,14 @@ project(Cppcheck)
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+# see : https://crascit.com/2016/04/09/using-ccache-with-cmake/
+find_program(CCACHE_PROGRAM ccache)
+if(CCACHE_PROGRAM)
+  set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
+else()
+  message("ccache not found, repeated builds will be faster if this program is installed")
+endif()
+
 include(GNUInstallDirs)
 
 include(cmake/versions.cmake)
-- 
2.17.1

