From 92a664e6116f2466721f7f080fc020ec9bab2137 Mon Sep 17 00:00:00 2001
From: Eric Squires <eric.squires@gtri.gatech.edu>
Date: Tue, 28 Sep 2021 11:10:52 -0400
Subject: [PATCH] adjust install location

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 2cb9efb..7d99dbb 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
-gitexecdir = ${DESTDIR}${shell git --exec-path}
-gitmanpath = ${DESTDIR}${shell git --man-path}
+gitexecdir = /usr/bin
+gitmanpath = ${shell git --man-path}
 
 -include ./config.mak
 
-- 
2.17.1

